content/derisking/index.html

```diff
@@ -0,0 +1,95 @@
+---
+title: De-risking government technology
+permalink: /derisking/
+layout: layouts/wide
+tags: derisking
+eleventyNavigation:
+  key: derisking-home-index
+  parent: derisiking-home
+  order: 1
+  title: Derisking Guide
+---
+
+<section class="usa-section--dark hero padding-y-6">
+  <div class="grid-container">
+    <div class="grid-row grid-gap flex-align-center">
+      <div class="tablet:grid-col-6">
+        <h1 class="tablet:font-heading-2xl section__heading--white margin-y-0">Deliver successful technology projects</h1>
+        <p class="font-heading-lg line-height-sans-3">Guidance for delivering successful custom technology projects in government.</p>
+        <a class="usa-button usa-button--accent margin-bottom-105" href="#guide-teasers">Read the guides</a>
+        <a class="usa-button usa-button--outline usa-button--inverse" href="#workshops">Request a workshop</a>
+      </div>
+      <div class="tablet:grid-col-6 display-flex tablet:flex-justify-center margin-top-2">
+        {% image_with_class "assets/derisking/img/guide-covers-circle.png" "maxw-mobile" "guide covers" %}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="usa-graphic-list usa-section usa-prose padding-bottom-0">
+  <div class="grid-container">
+    <div class="usa-graphic-list__row grid-row grid-gap">
+      <div class="usa-media-block">
```

Comment:
The pie chart is hidden on mobile width on the original page. Hiding the pie chart also makes the text column more readable on mobile.